spring-r2dbc/src/main/java/org/springframework/r2dbc/core/DatabaseClient.java

```diff
@@ -146,6 +149,12 @@ interface Builder {
 		 */
 		Builder namedParameters(boolean enabled);
 
+		/**
+		 * Handle the given error on {@link ConnectionAccessor#inConnection}
+		 * or {@link ConnectionAccessor#inConnectionMany}.
+		 */
+		Builder onConnectionError(BiConsumer<? super Throwable, ? super Connection> onConnectionError);
```

Comment:
```suggestion
		/**
		 * Handle an exception thrown from {@link ConnectionAccessor#inConnection}
		 * or {@link ConnectionAccessor#inConnectionMany}.
		 * @param errorHandler a {@code BiConsumer} that handles the connection error
		 * @since 6.2
		 */
		Builder onConnectionError(BiConsumer<? super Throwable, ? super Connection> errorHandler);
```

Reply:
oh thanks a lot for detailed review!
after checking https://github.com/spring-projects/spring-framework/pull/31942#issuecomment-1877220128, I'll update PR.

Reply:
✅ updated! rename all `BiConsumer onConnectionError` to `errorHandler`.